ui.tests/test-module/specs/radiobutton/radiobutton.runtime.spec.js

```diff
@@ -127,7 +127,7 @@ describe("Form with Radio Button Input", () => {
         const [radioButton2, radioButton2FieldView] = Object.entries(formContainer._fields)[1];
         const [radioButton4, radioButton4FieldView] = Object.entries(formContainer._fields)[3];
 
-        cy.get(`#${radioButton1}`).find("input").check("1").blur().then(x => {
+        cy.get(`#${radioButton1}`).find("input").check("1").then(x => {
```

Comment:
We should not remove blur from here ? It worked earlier and it should sill work without your changes.